pkg/apiserver/handlers.go

```diff
@@ -188,11 +190,11 @@ func (t *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	done := make(chan struct{}, 1)
+	done := make(chan struct{})
 	tw := newTimeoutWriter(w)
 	go func() {
 		t.handler.ServeHTTP(tw, r)
```

Comment:
We can worry about that separately, it doesn't need to block this PR :)

On Tue, Aug 9, 2016 at 11:37 AM, Xiang Li <notifications@github.com> wrote:

> In pkg/apiserver/handlers.go
> <https://github.com/kubernetes/kubernetes/pull/29594#discussion_r74118789>
> :
>
> > @@ -188,11 +190,11 @@ func (t *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
> >  		return
> >  	}
> >
> > -	done := make(chan struct{}, 1)
> > +	done := make(chan struct{})
> >  	tw := newTimeoutWriter(w)
> >  	go func() {
> >  		t.handler.ServeHTTP(tw, r)
>
> @lavalamp <https://github.com/lavalamp> OK. If you want a really panic on
> HTTP sever, I can help you fix it... But I will let you confirm that is
> what you want first...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29594/files/b9129837e9db2272de6f73ba88552451ae1438de#r74118789>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAngllAkcd926RDDAOmUu-LfPh95Jl9sks5qeMjUgaJpZM4JUwp4>
> .
>